# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pcmcia-cs/rc.pcmcia.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./etc/install-etc.orig	2003-01-18 08:38:22.000000000 +0100
+++ ./etc/install-etc	2003-01-18 12:00:46.000000000 +0100
@@ -50,6 +50,14 @@
 
 install_sysv ()
 {
+	cp rc.pcmcia /etc/rc.d/init.d/pcmcia
+	chmod +x /etc/rc.d/init.d/pcmcia
+	echo "The script 'pcmcia' has the priority 04." \
+		> /etc/rc.d/rcX.d/X04pcmcia
+}
+
+install_sysv_orig ()
+{
     SYSV=$PREFIX$RC_DIR/init.d/pcmcia
 
      if [ -d /etc/sysconfig ] ; then
--- ./etc/rc.pcmcia.orig	2002-08-08 08:43:43.000000000 +0200
+++ ./etc/rc.pcmcia	2003-01-18 08:38:22.000000000 +0100
@@ -1,4 +1,7 @@
 #!/bin/sh
+#
+# Desc: loading the PCMCIA drivers
+#
 
 # rc.pcmcia 1.45 2002/08/08 06:43:43 (David Hinds)
 #
@@ -41,7 +44,7 @@
     # Put pcmcia_core options here
     CORE_OPTS=
     # Put cardmgr options here
-    CARDMGR_OPTS=
+    CARDMGR_OPTS=-f
     # To set the PCMCIA scheme at startup...
     SCHEME=
 fi
